### PR TITLE
Added developer section to build script to fix deployment to Sonatype.

### PR DESCRIPTION
**Pull request by sdemchenko**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,14 +17,14 @@ Example for Maven:
 <dependency>
     <groupId>com.nfl.graphql</groupId>
     <artifactId>mediator</artifactId>
-    <version>1.0.2</version>
+    <version>x.y.z</version>
```
Comment (nflearl) [inline README.md]: Why can't these be driven by environment variables like the README was originally set up?

File: build.gradle [hunk 2]
```diff
@@ -86,7 +86,12 @@ publishing {
                         }
                     }
                 }
-
+                developers {
```
Comment (nflearl) [inline build.gradle]: Is this really necessary?  I don't care about credit per se.

Comment (sdemchenko) [reply build.gradle]: It seems that Bintray fails at POM validation stage due to missing 'developers' section.

Comment (nflearl) [reply build.gradle]: Good answer.

File: README.md (hunk 1 above)
Comment (sdemchenko) [reply README.md]: I have checked Git history and as fa as I can tell the version was either 'x.y.z' or '1.0.0'/'1.0.1'/'1.0.2' in this file.
I looked for a solution, but cannot find a way to read the version from the environment or file.
Glitr uses  'x.y.z', too.

**Pull request merged**